packages/cli/test/integration/users.api.test.ts

```diff
@@ -18,14 +18,11 @@ import * as testDb from './shared/testDb';
 import type { SuperAgentTest } from 'supertest';
 import type { Role } from '@db/entities/Role';
 import type { User } from '@db/entities/User';
-import { License } from '@/License';
-import { mockInstance } from '../shared/mocking';
 
-const testServer = utils.setupTestServer({ endpointGroups: ['users'] });
-
-const license = mockInstance(License, {
-	isAdvancedPermissionsLicensed: jest.fn().mockReturnValue(true),
-	isWithinUsersLimit: jest.fn().mockReturnValue(true),
+const testServer = utils.setupTestServer({
+	endpointGroups: ['users'],
+	enabledFeatures: ['feat:advancedPermissions'],
+	quotas: { 'quota:users': -1 },
```

Comment:
You shouldn't need this as it's the default. The reason `isWithinUsersLimit` was mocked was because I was replacing the entire instance and it's required for some part of the auth/routing stack.

Reply:
Good catch!